packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -52,10 +60,22 @@ const NominationBanner = ({ nomination }) => {
               </span>
             </div>
             <div className="column name">
-              <button className=" decline-button" onClick={declineApplication} style={{ background: '#f72314', color: '#ffffff', border: '#929292', fontWeight: 'bold' }}>
+              <button className=" decline-button" onClick={toggleModalState} style={{ background: '#f72314', color: '#ffffff', border: '#929292', fontWeight: 'bold' }}>
                 Decline Application
               </button>
             </div>
+            <div className={`modal-background modalShowing-${modalState}`}>
```

Comment:
Let's see if we can use conditional rendering to show/hide the component https://reactjs.org/docs/conditional-rendering.html